pkg/api/controllers/metrics.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2019 Huawei Technologies Co., Ltd. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+This module implements a entry into the OpenSDS northbound service.
+
+*/
+
+package controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	log "github.com/golang/glog"
+	"github.com/opensds/opensds/pkg/api/policy"
+	c "github.com/opensds/opensds/pkg/context"
+	"github.com/opensds/opensds/pkg/controller/client"
+	"github.com/opensds/opensds/pkg/model"
+	pb "github.com/opensds/opensds/pkg/model/proto"
+	. "github.com/opensds/opensds/pkg/utils/config"
+	"golang.org/x/net/context"
```

Comment:
Please change `golang.org/x/net/context` to `context`.

Reply:
done